srcpkgs/pyliblo3/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'pyliblo3'
+pkgname=pyliblo3
```

Comment:
This package should be called `python3-pyliblo3`.

Reply:
I followed the suggestion and updated the package name to `python3-pyliblo3`.